sys-tuner/src/main.rs

```diff
@@ -65,10 +75,14 @@ fn main() {
     let listener = unix_socket::UnixListener::bind(solana_sys_tuner::SOLANA_SYS_TUNER_PATH)
         .expect("Failed to bind to the socket file");
 
+    #[cfg(target_os = "linux")]
```

Comment:
The `users` crate works for macos right?  If so can we drop this linux cfg and run `users::get_user_by_name()` for  all of cfg(unix)?